requirements.txt

```diff
@@ -14,3 +14,4 @@ Flask-Script==2.0.5
 workdays==1.4
 unicodecsv==0.14.1
 cryptography==1.6
+git+https://github.com/alphagov/notifications-python-client.git@4.0.0#egg=notifications-python-client==4.0.0
```

Comment:
@allait @TheDoubleK @benvand we use it in python 2.7 in our load tests, which are fairly informal, but do work. the only steps we need to take to get it on pypi is run tests against it and then update the meta tags to support it. Zero code change at all is anticipated, and after we're released I don't plan on dropping support for [65% of the python userbase](https://twitter.com/vlasovskikh/status/801720613312364544?ref_src=twsrc%5Etfw) 😬

[the ticket's in the backlog](https://www.pivotaltracker.com/story/show/137108317) and i do plan on working on that Soon™